### ISSUE TITLE
BBC TV content not downloading with Overplay 

### ISSUE THREAD

**Issue by naq59** [GetiPlayerAutomator/get-iplayer-automator#580]
Since last week I can´t download any BBC TV content using Overplay Smart DNS. Radio and ITV are fine as is watching BBC TV online via iPlayer. This issue was in reported under #576. I´ve written several times to Overplay but had no reply so far. Mac OSX 10.11.5 and 10.10.5 GiA 1.8.7 pre3 and 1.8.6. It doesn´t matter what BBC TV programme I select or how I load it into GiA. Maybe it is an Overplay issue BUT I noticed someone had exactly the same problem last year in case #476 but I can´t read how the matter was solved. GiA doesn´t crash it just keeps on trying to initialise until it gives up and reports unknown failure. I´ve included the Verbose Log link hopefully correctly.
I would be very grateful for any advice.
[BBC TV issue.txt](https://github.com/GetiPlayerAutomator/get-iplayer-automator/files/366044/BBC.TV.issue.txt)

**Comment by dinkypumpkin** [comment]
There is nothing GiA can do about your problems with Overplay, nor is there anything it can do to prevent Overplay - and by extension, you - from being blocked by the BBC.

Duplicate of #576

**Comment by naq59** [comment]
Thanks for the prompt reply. Is there any way to find out if or how case #476 was solved as it sounds more or less identical?

naq59

**Comment by dinkypumpkin** [comment]
No

**Comment by larryy** [comment]
naq59 - I don't know anything about Overplay, but it is likely the BBC has simply blocked it, in which case there's nothing GiA can do about it, as dinkypumpkin noted.  I strongly suspect that if you go to the iPlayer website in a browser while using Overplay you will see that the content is blocked.  In such a situation, it's your access which is blocked, having nothing to do with GiA, which is why dinkypumpkin is so abrupt in dismissing the issue.  Also, these days at least, neither GiA nor get_iplayer developers will actively support or even discuss non-UK use of the tools.

**Comment by naq59** [comment]
I can watch BBC TV online via iPlayer using Overplay as well as download ITV and BBC radio so my access is there. I´ve just tried Unlocator and the issue is identical so it´s probably not Overplay or Unlocator but something BBC has done. This started 8-9 days ago and affects all my macs, even with different internet providers also using Chrome instead of Safari.
 I can´t imagine I´m the only person using this combination and having problems
naq59

**Comment by Vangelis66** [comment]
@naq59 wrote: 

> I can watch BBC TV online via iPlayer using Overplay

... Just to clear up  a very frequent misconception here, 
the very fact you can currently still watch BBC iPlayer in 
a browser from overseas (utilising your SmartDNS service) 
**has nothing to do** with your (in)abilitity to download 
BBC TV programmes via GiA or, to be more precise, 
with get_iplayer perl script (GiP, which is the component of 
GiA responsible for BBC downloads). 
BBC iPlayer in a desktop browser uses by default AdobeHDS 
streams (over HTTP), GiA/GiP have no support for those!

(If I am not mistaken, GiA-released-version still uses GiP v2.94, 
which in turn relies upon the older RTMP streams for downloading; 
recently released (as standalone) GiP v2.95 has changed to 
AppleHLS streams for both TV & radio, but again those are not 
the streams you can access in a browser. 
If you are an owner of an iPad, you can try to configure it 
to use Overplay and then check whether you are able to 
access the HLS streams served to that - if affirmative, maybe in the 
future when GiP v2.95 is intregrated into the next GiA release - 
and provided it implements support for HLS, see #513 - you'll 
have better luck... )

> as well as download ITV

No connection whatsoever with the (in)abilitity to download BBC TV... 

> and BBC radio

While BBC radio is still geo-filtered, **it is not fully geo-blocked**; 
i.e. if your SmartDNS fails the BBC geo-checks, you'd still get a download, 
be it in the lower international bitrates...

> but something BBC has done.

I am not a conspiracy theorist myself, but be certain BBC staff 
are actively monitoring internet places like this one, where 
overseas access to iPlayer is admitted to; more than a year ago 
they started blacklisting first open HTTP proxies, then moved on 
to commercial VPNs/Proxy providers, now it's the time for 
SmartDNS services. Expect things to become even tougher 
come next September, when a new law is put into force that 
requires a TV licence (obviously in the UK) even for on-demand 
TV iPlayer content!

But @larryy is right: 

> neither GiA nor get_iplayer developers will actively support or even discuss non-UK use

According to the recent wiki entry:
https://github.com/GetiPlayerAutomator/get-iplayer-automator/wiki/No-Support-for-GiA-Outside-the-UK
you shouldn't have started this issue in the first place; 
but then again it was created after your issue 
and quite possibly as a result of your issue: 

> It is likely that you will still be able to access iPlayer and ITV Hub 
> with your VPN/VPS/proxy in your web browser when GiA is blocked. 
> GiA works differently, so do not think that what happens in your browser 
> is evidence that GiA should work - it is not.

Kind regards

**Comment by naq59** [comment]
Thanks for this very comprehensive answer . I understand this issue better now hope all is not lost for the future. By the way, is there an alternative for downloading BBC TV without DRM? I'm fine paying for a UK license and I just would like to watch programmes later and/or again.

Regards
naq 59
